### PR TITLE
Update ui.py to refer to Python

### PR DESCRIPTION
**Pull request by sfc-gh-ghernandez**

### CODE REVIEW

**Comment by sfc-gh-cgorrie** [comment]
We are migrating this repo to another location -- I'll get in touch with you to get this merged at the new location when it's ready. Thanks!

**Comment by sfc-gh-ghernandez** [comment]
> We are migrating this repo to another location -- I'll get in touch with you to get this merged at the new location when it's ready. Thanks!

Thanks. Do you think we could merge this just for tomorrow's demo that I'm doing? Beyond that I won't be using the repo

File: streamlit-python/src/module-ui/src/ui.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # Write directly to the app
 st.title("Hello Snowflake!")
 st.write(
-   """The sum of the two numbers are calculated by the Java Add.two() function
+   """The sum of the two numbers is calculated by the Python add_fn() function
```
Comment (sfc-gh-cgorrie) [inline streamlit-python/src/module-ui/src/ui.py:8]: It's called `core.add()`

Comment (sfc-gh-ghernandez) [reply streamlit-python/src/module-ui/src/ui.py:8]: In the setup sql file, but in the core file isn't it add_fn()? I can update it, I was just trying to follow the Java template README as an example without over-wordsmithing this :D

Comment (sfc-gh-ghernandez) [reply streamlit-python/src/module-ui/src/ui.py:8]: By "core file" I mean streamlit-python/src/module-add/src/main/python/add.py

**Review by sfc-gh-cgorrie — APPROVED** [review]

**Pull request merged**